packages/image_picker/image_picker_platform_interface/lib/src/types/picked_file/base.dart

```diff
@@ -52,7 +52,7 @@ abstract class PickedFileBase {
   /// If `end` is present, only up to byte-index `end` will be read. Otherwise, until end of file.
   ///
   /// In order to make sure that system resources are freed, the stream must be read to completion or the subscription on the stream must be cancelled.
-  Stream<Uint8List> openRead([int start, int end]) {
+  Stream<Uint8List> openRead([int? start, int? end]) {
```

Comment:
Should this be named parameters instead?  `openRead(end: 10)` is better than `openRead(null, 10)`

Reply:
These follow [the File API](https://api.dart.dev/stable/2.10.5/dart-io/File/openRead.html). Anyway, this code will eventually be removed (it's available, and reusable in the cross_file package now)

Reply:
I think it's unrelated to NNBD and I'd prefer not to break the API. Also it's going to be removed soon (probably the very next patch), let's not introduce 2 consecutive breaking changes for people to migrate.

Reply:
sgtm